src/Reflection/ParametersAcceptorSelector.php

```diff
@@ -515,6 +515,10 @@ private static function getCurlOptValueType(int $curlOpt): ?Type
 			return new UnionType([new ConstantIntegerType(0), new ConstantIntegerType(2)]);
 		}
 
+		if (defined('CURLOPT_POSTFIELDS') && $curlOpt === CURLOPT_SSL_VERIFYHOST) {
```

Comment:
Copy paste error most likely here. Would be nice to verify also when a wrong type is passed when setting `CURLOPT_POSTFIELDS`.